web/templates/web/study-detail.html

```diff
@@ -138,7 +138,7 @@ <h4>{% trans "Would you like to participate in this study?" %}</h4>
                         {% if not request.user.is_authenticated %}
                             <a class="btn btn-lg btn-default" href="{% url 'login' %}">{% trans "Log in to participate" %}</a>
                         {% elif not children %}
-                            <a class="btn btn-lg btn-default" href="{% url 'web:children-list' %}">{% trans "Add child profile to " %} {% if preview_mode %} {% trans "preview" %} {% else %} {% trans "participate" %} {% endif %} %}</a>
+                            <a class="btn btn-lg btn-default" href="{% url 'web:children-list' %}">{% trans "Add child profile to " %} {% if preview_mode %} {% trans "preview" %} {% else %} {% trans "participate" %} {% endif %}</a>
```

Comment:
Thank you!! Looks like this will also close https://github.com/lookit/lookit-api/issues/666 :)